frontend/src/sections/profile/ProfileEditForm.tsx

```diff
@@ -117,9 +117,14 @@ export default function ProfileEditForm({ isExistingUser, buttonText }: Props) {
               />
               <FormHelperText error>{errors?.telegramHandle?.message}</FormHelperText>
               <RHFTextField
+                disabled={isExistingUser}
                 name="telegramHandle"
                 label="Telegram Username"
-                helperText="Other climbers will communicate with you over Telegram."
+                helperText={
+                  isExistingUser
```

Comment:
Actually I just remembered, isExistingUser is always true. B/c I made a quick and dirty fix for onboarding form, to turn it true. So that, the OAuth data will auto-populate. 

I think in that case, the new message won't make sense. I suggest discussing with @CrownKira who's working on the new onboarding for the way forward.

Reply:
Another solution is you could add another prop name. I don't think this is an urgent issue for now.